scipy/ndimage/interpolation.py

```diff
@@ -211,17 +211,30 @@ def geometric_transform(input, mapping, output_shape=None,
 
     Examples
     --------
-    >>> from scipy import ndimage
+    >>> import numpy as np
+    >>> from scipy.ndimage import geometric_transform
     >>> a = np.arange(12.).reshape((4, 3))
     >>> def shift_func(output_coords):
     ...     return (output_coords[0] - 0.5, output_coords[1] - 0.5)
     ...
-    >>> ndimage.geometric_transform(a, shift_func)
+    >>> geometric_transform(a, shift_func)
     array([[ 0.   ,  0.   ,  0.   ],
            [ 0.   ,  1.362,  2.738],
            [ 0.   ,  4.812,  6.187],
            [ 0.   ,  8.263,  9.637]])
-
+    >>> b = [1, 2, 3, 4, 5]
```

Comment:
Two tiny revisions (just to make it look pretty): add a blank line between the two examples so that they don't run together:

https://4828-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.ndimage.geometric_transform.html#scipy.ndimage.geometric_transform

Also should probably also call the second `shift` function `shift_func` too just so that it's consistent.

Reply:
Fixed.